README.md

```diff
@@ -65,23 +82,29 @@ where `message_board` is the name of the canister.
   "version": "0.1.0",
   "description": "Internet Computer message board application",
   "dependencies": {
-    "@dfinity/agent": "^0.19.2",
-    "@dfinity/candid": "^0.19.2",
-    "azle": "^0.18.6",
-    "uuid": "^9.0.0"
+    "@dfinity/agent": "^0.21.4",
+    "@dfinity/candid": "^0.21.4",
+    "azle": "^0.20.1",
+    "body-parser": "^1.20.2",
```

Comment:
You shouldn't need to install body-parser, express.json() or express.text() just works